Code/Framework/AzFramework/CMakeLists.txt

```diff
@@ -25,13 +26,20 @@ ly_add_target(
             ${pal_dir}
         PRIVATE
             ${common_dir}
+            AZ::AzNetworking
```

Comment:
This actually needs to be removed.
The `INCLUDE_DIRECTORIES` option accepts an actual directory.